source/frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsContainer.tsx

```diff
@@ -4,7 +4,8 @@ import useIsMounted from 'hooks/useIsMounted';
 import { useQueryMapLayersByLocation } from 'hooks/useQueryMapLayersByLocation';
 import isNumber from 'lodash/isNumber';
 import { Api_Property } from 'models/api/Property';
-import React, { useEffect, useRef, useState } from 'react';
+import React from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
nit: why the duplicate import lines? was this auto-generated by the linter? 
I would expect all imports from same package/library to be on a single line...